src/Elm/Syntax/Exposing.elm

```diff
@@ -134,10 +134,24 @@ decoder : Decoder Exposing
 decoder =
     decodeTyped
         [ ( "all", Range.decoder |> JD.map All )
-        , ( "explicit", JD.list topLevelExposeDecoder |> JD.map Explicit )
+        , ( "explicit", decodeNonemptyList topLevelExposeDecoder |> JD.map (\( head, rest ) -> Explicit head rest) )
         ]
 
 
+decodeNonemptyList : Decoder a -> Decoder ( a, List a )
```

Comment:
This function is duplicated from Expression.elm since I didn't want to expose that function to users. I'm guessing this all will get removed anyway once we start using elm-codec.